py/dml/codegen.py

```diff
@@ -104,18 +104,26 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         assert LoopContext.current is self
         LoopContext.current = self.prev
+    def break_(self, site):
+        raise ICE(site, f'{__class__.__name__}.break_ not implemented')
+    def cont_(self, site):
```

Comment:
`continue_`?
also, if this didnt cause a ECONTU test failure then the test should be fixed

Reply:
Oops! `cont_` is a bad old artifact.
And no, I just forgot to run `1.2/errors/ECONTU` and only ran the 1.4 tests; so it's not a test gap.